configcatclient/configcache.py

```diff
@@ -4,10 +4,10 @@
 class InMemoryConfigCache(ConfigCache):
 
     def __init__(self):
-        self._value = None
+        self._value = {}
 
-    def get(self):
-        return self._value
+    def get(self, key):
+        return self._value.get(key, None)
```

Comment:
`None` is unnecessary here because this is the default value of get: `dict.get(key, default=None)`
Please remove it.

Reply:
done